### PR TITLE
Everything Events Branch

### PR DESCRIPTION
**Pull request by Sarai-ii**

### CODE REVIEW

File: src/components/CurrentEvent.js [hunk 1]
```diff
@@ -53,6 +54,7 @@ export default function CurrentEvent({event, userData, userId}) {
       <div className="container">
         <div className="description-CE-container">
           <h5 className="description-price-title"><span className="price-title">Minimum Spend:</span><span className="price">${event.minimum_spend}</span></h5>
+          <h5 className="description-price-title"><span className="price-title">Maximum Spend:</span><span className="price">${event.minimum_spend + 30} </span></h5>
```
Comment (JenniferEinstein) [inline src/components/CurrentEvent.js:57]: Why are you adding a maximum spend here? We already decided not to do this.

**Review by JenniferEinstein — CHANGES_REQUESTED** [review]
There should not be a maximum spend here. There is no reason for it and we have discussed it. 

Also, there are copy writing changes that should be made to the events sign up page, chief of which is all instances of i.e. should be replaced with e.g. Let me knowif it would be more helpful to have a list in slack or in here.

File: src/components/CurrentEvent.js (hunk 1 above)
Comment (Sarai-ii) [reply src/components/CurrentEvent.js:55]: Because That's what made sense.

**Comment by Sarai-ii** [comment]
Merged myself by accident I thought it was committing based on the review.

**Pull request merged**